tools/mount_gcsfuse/main.go

```diff
@@ -84,7 +84,8 @@ func makeGcsfuseArgs(
 			"disable_http2",
 			"experimental_local_file_cache",
 			"experimental_enable_storage_client_library",
-			"reuse_token_from_url":
+			"reuse_token_from_url",
+			"enable-nonexistent-type-cache":
```

Comment:
We need to change the hyphen with underscore in this flag. Since, while mounting through fstab entry we pass the flag as underscore.

For example,
fstab entry: my-bucket /mount/point gcsfuse rw,noauto,user,key_file=/path/to/key.json

As you can see, we have provided the flag (key_file) with underscore not hyphen.

Reply:
I see. Thanks for the hints!